torch/_inductor/codecache.py

```diff
@@ -575,79 +588,193 @@ def __init__(self, fx_args: List[Any], fx_kwargs: Dict[str, Any]) -> None:
         self.inductor_config = config.save_config()  # type: ignore[attr-defined]
         self.inductor_code_hash = get_inductor_code_hash()
 
+    def debug_str(self):
+        """
```

Comment:
This method is helpful for printing all the fields that we include in the hash when computing the cache key. It is not currently called anywhere, however, because I think it's waaaay too verbose even for debug logging. Does anyone know if we have precedent for different log levels?

Reply:
There is log.info vs log.debug. Debug is more verbose

Reply:
Yeah, I'm saying that even log.debug is probably too verbose here. I think you only want to print this level of detail if you need to deeply debug a cache miss that you think shouldn't be a cache miss and you want to see all the gory details of what went into the key calculation.

Reply:
You could enable it with an environment variable

Reply:
Don't do an environment variable. Instead, you can create an artifact logger that isn't triggered even in debug. bytecode is done this way, check it out.

Reply:
@ezyang.  Sry, I don't know what bytecode refers to. Is this the sample you have in mind? https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/convert_frame.py#L85